content/doc/developer/plugin-development/incrementals.adoc

```diff
@@ -32,12 +32,7 @@ For more details, please refer to the link:https://github.com/jenkinsci/incremen
 == Enabling Incrementals
 
 Inside the (downstream) plugin, which is supposed to be the API consumer, you can enable incrementals by running:
-```
-mkdir -p .mvn
-echo -Pconsume-incrementals >> .mvn/maven.config
-```
-
-link:https://github.com/jenkinsci/incrementals-tools[Another way] is to run `mvn incrementals:incrementalify`.
+`mvn incrementals:incrementalify`.
```

Comment:
Just in case, are the setup instructions elsewhere setting up Maven in the instructions to get access to the `incrementals` plugin?

Reply:
It’s handled by the parent Pom, this guide also mentions it’s a quick start and full docs are in incremental-tools repo

Reply:
Right, I'm asking in case the first step would otherwise fail. I remember advice to add

```
  <pluginGroups>
    <pluginGroup>org.jenkins-ci.tools</pluginGroup>
  </pluginGroups>
```

or similar to `settings.xml`, hence the question. But I guess the reference to `incrementals-maven-plugin` takes care of that 👍

Reply:
not required after adding it to the parent pom, although you do get a build warning if you don't have that in your settings.xml